app/test_main.py

```diff
@@ -1,3 +1,25 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize("cat_age, dog_age, expected", [
+    (0, 0, [0, 0]),
+    (14, 14, [0, 0]),
+    (15, 15, [1, 1]),
+    (23, 23, [1, 1]),
+    (24, 24, [2, 2]),
+    (27, 28, [2, 2]),
+    (28, 29, [3, 3]),
+    (100, 100, [21, 17]),
+])
+def test_get_human_age(
+        cat_age: int,
+        dog_age: int,
+        expected: list
+) -> None:
+    if expected == "Invalid input":
+        with pytest.raises(ValueError):
+            get_human_age(cat_age, dog_age)
```

Comment:
To improve test organization and reusability, consider creating a separate function specifically for testing type errors in 'get_human_age' function.